content/docs/studio/how-it-works.md

```diff
@@ -1,48 +1,58 @@
 # How Iterative Studio Works
 
-[Iterative Studio](https://studio.iterative.ai/) works with the data, metrics,
-hyperparameters and model metadata that you add to your ML project Git
-repositories. It works very closely with your Git ecosystem.
+[Iterative Studio](https://studio.iterative.ai/) works very closely with your
+Git ecosystem. It parses the data, metrics, hyperparameters and model metadata
+that you add to your ML project Git repositories, and renders them on a web UI.
 
-This video illustrates how Iterative Studio works closely with your Git
-ecosystem.
+Additionally, you can [track metrics from your experiments in real
+time][live-metrics-and-plots] by pushing the metrics directly to Iterative
+Studio using [DVCLive].
+
+The following two videos illustrate how Iterative Studio works with data from
+your ML projects and experiments.
```

Comment:
Minor but I'm also not sure we want 2 video embeds one after another here.

We have something similar here: https://dvc.org/doc/user-guide/experiment-management/experiments-overview#work-with-dvc-experiments-from-a-gui and it's a bit strange.